developer_manual/app_publishing_maintenance/release_process.rst

```diff
@@ -138,3 +138,38 @@ Prepare follow-up releases
 ~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The target milestone was closed in the release preparation. Now it's time to create a new milestone for the next release(s).
+
+
+Shipped Apps
+------------
+
+The majority of apps is distributed via the Nextcloud app store. A few apps are bundled and shipped with Nextcloud. There are a few things to keep in mind for them.
+
+Git branch management
+~~~~~~~~~~~~~~~~~~~~~
+
+The release script simply git-clones app repositories. Repositories of shipped apps need branches to correspond to the branches in the `Nextcloud server repository <https://github.com/nextcloud/server>`_:
```

Comment:
Should mention that composer dependencies and JS assets need to committed in the repo?